src/components/App.tsx

```diff
@@ -2576,6 +2609,26 @@ class App extends React.Component<ExcalidrawProps, AppState> {
     return false;
   };
 
+  private isHittingASelectedElement(
+    hitElement: ExcalidrawElement | null,
+  ): boolean {
+    return hitElement != null && this.state.selectedElementIds[hitElement.id];
+  }
+
+  private isShiftKeyPressed(
+    event: React.PointerEvent<HTMLCanvasElement>,
+  ): boolean {
+    return event.shiftKey;
+  }
+
+  private isHittingCommonBoundingBoxOfSelectedElements(
+    point: Readonly<{ x: number; y: number }>,
+    selectedElements: readonly ExcalidrawElement[],
+  ): boolean {
+    const [x1, y1, x2, y2] = getCommonBounds(selectedElements);
+    return point.x > x1 && point.x < x2 && point.y > y1 && point.y < y2;
+  }
```

Comment:
Should use threshold same as single elements.

Reply:
Added threshold.